src/OagBundle/Resources/public/sass/main.scss

```diff
@@ -3,92 +3,201 @@
 
 @import 'vars';
 
-body {
-    background: #eeeeee;
-    margin: 15px;
-    font-family: $oag_font_copy;
+
+/* --------------------------------------------------------------
+
+   Page Layout
+
+-------------------------------------------------------------- */
+
+html, body {
+  height: 100%;
+  font-family: $oag_font;
+  font-size: 1em;
+  line-height: 1.5;
+  color: $oag_dark;
+  margin: 0;
+  background: #efefef;
 }
 
-.associated-files .docname {
-    margin-top: 5px;
-    width: 250px;
-    display: inline-block;
-    border-bottom: 1px dotted;
+h1 {
+  font-size: 2.5em;
 }
 
-.hidden-row {
-    display: none;
+h2 {
+  font-weight: normal;
+}
+
+hr {
+  margin-bottom: 2em;
+}
+
+a:-webkit-any-link {
+  color: $oag_dark;
+}
+
+a {
+  text-decoration: none;
+  &:active,
+  &:visited {
+    text-decoration: underline;
+  }
+  &:hover {
+    color: $oag_link;
+    text-decoration: underline;
+  }
+}
+
+// button {
```

Comment:
Can we remove or are you planning on keeping for something else?

Reply:
Keeping for later. :+1: